specs/src/specs/data_structures.md

```diff
@@ -72,12 +72,11 @@ Implementations can prune rows containing only [tail padding](./consensus.md#res
 
 Data that is [erasure-coded](#erasure-coding) for [data availability checks](https://arxiv.org/abs/1809.09044).
 
-| name                        | type                                                    | description                                                                                                  |
-|-----------------------------|---------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|
-| `transactionData`           | [TransactionData](#transactiondata)                     | Transaction data. Transactions modify the validator set and balances, and pay fees for blobs to be included. |
-| `intermediateStateRootData` | [IntermediateStateRootData](#intermediatestaterootdata) | Intermediate state roots used for fraud proofs.                                                              |
-| `payForBlobData`            | [PayForBlobData](#payforblobdata)                       | PayForBlob data. Transactions that pay for blobs to be included.                                             |
-| `blobData`                  | [BlobData](#blobdata)                                   | Blob data. Blobs are app data.                                                                               |
+| name                        | type                                                    | description                                                                                                           |
+|-----------------------------|---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| `transactions`              | [Transaction](#transaction)                             | Transactions are ordinary Cosmos SDK transactions. For example: they may modify the validator set and token balances. |
+| `payForBlobData`            | [PayForBlobData](#payforblobdata)                       | PayForBlob data. Transactions that pay for blobs to be included.                                                      |
+| `blobData`                  | [BlobData](#blobdata)                                   | Blob data. Blobs are app data.                                                                                        |
```

Comment:
```Blobs are app data```

This feels like an incorrect description

Reply:
The specs use "app data" to refer to data published by applications on top of Celestia (like roll-ups).

Reply:
dc14f8b027aed885cbcd4067632eaab3ae5980b7